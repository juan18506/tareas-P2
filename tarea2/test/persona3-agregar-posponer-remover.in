crearAgenda

crearPersona 1 20 Persona1

# creo cuatro eventos para ingresarlos a la agenda de la persona

crearFecha 22/3/2023
crearEvento 1 Descripcion uno
agregarEventoPersona

crearFecha 23/3/2023
crearEvento 2 Descripcion dos
agregarEventoPersona

crearFecha 24/3/2023
crearEvento 3 Descripcion tres
agregarEventoPersona

crearFecha 1/4/2023
crearEvento 4 Descripcion cuatro
agregarEventoPersona

imprimirAgendaPersona

# posponer evento 1 tres dias
posponerEventoPersona 1 3

imprimirAgendaPersona

# remuevo el evento 2
removerEventoPersona 2

imprimirAgendaPersona

liberarPersona

Fin