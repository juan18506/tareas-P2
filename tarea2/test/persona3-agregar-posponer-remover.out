1>2>3>#  creo cuatro eventos para ingresarlos a la agenda de la persona.
4>5>6>7>8>9>10>11>12>13>14>15>16>Evento 1:  Descripcion uno
Fecha: 22/3/2023
Evento 2:  Descripcion dos
Fecha: 23/3/2023
Evento 3:  Descripcion tres
Fecha: 24/3/2023
Evento 4:  Descripcion cuatro
Fecha: 1/4/2023
17>#  posponer evento 1 tres dias.
18>19>Evento 2:  Descripcion dos
Fecha: 23/3/2023
Evento 3:  Descripcion tres
Fecha: 24/3/2023
Evento 1:  Descripcion uno
Fecha: 25/3/2023
Evento 4:  Descripcion cuatro
Fecha: 1/4/2023
20>#  remuevo el evento 2.
21>22>Evento 3:  Descripcion tres
Fecha: 24/3/2023
Evento 1:  Descripcion uno
Fecha: 25/3/2023
Evento 4:  Descripcion cuatro
Fecha: 1/4/2023
23>24>Fin.
